painttiming.bs

```diff
@@ -192,6 +195,10 @@ Reporting paint timing {#sec-reporting-paint-timing}
 
 <div algorithm="Mark paint timing">
     When asked to [=mark paint timing=] given a [=Document=] |document| as input, perform the following steps:
+    1. If the [=document=]'s [=browsing context=] |ctx| is a [=nested browsing context=] and paint-timing is not enabled for |ctx|, return.
```

Comment:
Hmm not super clear what 'paint-timing is not enabled' means. I wonder if we can have a more precise check here. Perhaps we can check whether 'paint' is in supportedEntryTypes?

Reply:
That would be a bit circular, as later we say that "paint" is not there when not supported...

Reply:
I've revised, better clarifying "eligibility"

Reply:
Awesome!